clients/bmp_client.go

```diff
@@ -23,8 +24,19 @@ type bmpClient struct {
 
 func NewBmpClient(username, password, url string, hClient softlayer.HttpClient, configPath string) *bmpClient {
 	var httpClient softlayer.HttpClient
+
+	useHttps := false
+	if url != "" {
```

Comment:
Cool, but move this to a utility public method in `common` named like `IsHttps(url string) bool` or similar